python/tests/test_python_serving_file.py

```diff
@@ -0,0 +1,33 @@
+from konduit import *
+from konduit.server import Server
+from konduit.client import Client
+from konduit.utils import is_port_in_use, default_python_path
+
+import numpy as np
+import os
+
+
+def test_python_script_prediction():
+    port = 1337
+    serving_config = ServingConfig(http_port=port)
```

Comment:
Generate a random untaken port for tests please.

Reply:
@maxpumperla

Reply:
@agibsonccc if the tests run properly, that's not necessary. the clean-up at the end will free up the port again. we could think about using Python's `UnitTest` to get proper `before` and `after` methods for tests, just like we have them in java.